python_modules/dagster/dagster_tests/cli_tests/workspace_tests/test_grpc_server_workspace.py

```diff
@@ -147,6 +147,23 @@ def test_ssl_grpc_server_workspace(instance):
             pass
 
 
+def test_port_range(instance):
+    with environ({"DAGSTER_PORT_RANGE": "12345-12399"}):
+        with GrpcServerProcess(
+            instance_ref=instance.get_ref(), force_port=True, wait_on_exit=True
+        ) as server_process:
+            assert server_process.port
+            assert server_process.port >= 12345
+            assert server_process.port <= 12399
+
+    with pytest.raises(Exception, match="DAGSTER_PORT_RANGE must be of fhe form"):
```

Comment:
of the form

I although I assume there's a failing test somewhere in the build to catch this

Reply:
not if you make the same typo in the real error message!